modules/nw-osp-pod-adding-connections-ipv6.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-osp-pod-adding-connections-ipv6_{context}"]
+= Adding IPv6 connectivity to pods on {rh-openstack}
+
+After you enable IPv6 connectivity in pods, add connectivity to them by using a Container Network Interface (CNI) configuration.
+
+.Procedure
+
+. To edit the Cluster Network Operator (CNO), enter the following command:
++
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+. Specify your CNI configuration under the `spec` field. For example, the following configuration uses a SLAAC address mode with MACVLAN:
++
+[source,yaml]
+----
+...
+spec:
+  additionalNetworks:
+  - name: ipv6
+    namespace: ipv6 <1>
+    rawCNIConfig: '{ "cniVersion": "0.3.1", "name": "ipv6", "type": "macvlan", "master": "ens4"}' <2>
+    type: Raw
+----
+<1> Be sure to create pods in the same namespace.
+<2> The interface in the network attachment `"master"` field can differ from `"ens4"` when more networks are configured or when a different kernel driver is used.
```

Comment:
Guessing that the use of `master` here is programmatic and required here, right?